web/src/redux/apis/TraceTest.api.ts

```diff
@@ -52,6 +54,29 @@ const TraceTestAPI = createApi({
       providesTags: () => [{type: Tags.TEST, id: 'LIST'}],
       transformResponse: (rawTestList: TTest[]) => rawTestList.map(rawTest => Test(rawTest)),
     }),
+    getTransactionById: build.query<ITransaction, {transactionId: string}>({
+      query: () => `/tests`,
+      providesTags: result => [{type: Tags.TRANSACTION, id: result?.id}],
+      transformResponse: () => ({
```

Comment:
Same as for envs, let's use the mocked models instead of raw response so we have a clear path for when we have to integrate the BE

Reply:
✅